src/sentry/models/file.py

```diff
@@ -238,9 +238,9 @@ def from_file(cls, fileobj):
         return blob
 
     @classmethod
-    def generate_unique_path(cls, timestamp):
-        pieces = [six.text_type(x) for x in divmod(int(timestamp.strftime('%s')), ONE_DAY)]
-        pieces.append(uuid4().hex)
+    def generate_unique_path(cls):
+        uuid_hex = uuid4().hex
```

Comment:
Can we leave a comment here that we're intentionally not using the checksum as the path name to avoid concurrency issues (permit concurrent writes temporarily). Otherwise someone will change that in the future and overlook it.